packages/telemetry/src/slog-to-otel.js

```diff
@@ -662,16 +871,17 @@ export const makeSlogToOtelKit = (tracer, overrideAttrs = {}) => {
         break;
       }
       case 'crank-start': {
-        const { message, ...crankAttrs } = slogAttrs;
-        const [name, messageAttrs, links] = extractMessageAttrs(message);
+        const { message, crankType, ...crankAttrs } = slogAttrs;
+        if (crankType === 'routing') {
+          break;
+        }
+        const [name, _messageAttrs, links] = extractMessageAttrs(message);
         spans.startNamed(
```

Comment:
I was hoping we'd be able to use `push` instead of `startNamed` to get rid of the notion of named spans with explicit parents, but I guess that can come later.

Reply:
I tried, I really did, but there's enough inexplicable weirdness in the difference between what `startNamed` and `push` do that it defeated me.